python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -40,6 +40,41 @@ def noao_section_keyword(bbox, flipx=False, flipy=False):
     return "[%i:%i,%i:%i]" % (xmin, xmax, ymin, ymax)
 
 
+def translate_comcam_filter(phosimFilter):
+    """
+    Translate phosim filter name to that appropriate
+    for obs_lsst - see DM-21706
+
+    Parameters
+    ----------
+    phosimFilter: str
+        Name of phosim filter in the phosim file header,
+        one of u,g,r,i,z,y
+
+    Returns
+    --------
+    mappedFilter: str
+        A translated filter name allowed by obs_lsst
+        for lsstComCam instrument
+    """
+    phosim_to_comcam = {
+        "u": "06",
+        "g": "01",
+        "r": "03",
+        "i": "06",
+        "z": "02",
+        "y": "04",
+    }
+    # test if filter can be translated
+    if phosimFilter in phosim_to_comcam.keys():
+        suffix = phosim_to_comcam[phosimFilter]
+        return f"{phosimFilter}_{suffix}"
+    else:
+        raise ValueError(
+            f"This phosim filter:{phosimFilter} cannot \
+            be translated for comcam")
```

Comment:
Please document the `ValueError` in function's doc string as well.

Reply:
Added documentation